### ISSUE TITLE
Outline missing on keyboard focus in Greymond theme (Thunar / file manager view)

### ISSUE THREAD

**Issue by Edguy-HC** [parhelion22/xfce-theme-greymond#1]
Hello,

I’d like to report a usability issue I discovered in the Greymond GTK theme, specifically when using Thunar on MX Linux.

When I move between files using the keyboard with Ctrl+Arrow, the visual outline that would indicate focus does not appear at all, neither on the item moved to, nor on selected items.
So currently there is no visible feedback when navigating by keyboard, which makes it hard to see which file has focus if it’s not already selected.

I solved the problem by adding this CSS block to the gtk.css file:
```
.flowboxchild:focus,
.iconview:focus,
.view:focus,
.flowboxchild:selected,
.iconview:selected,
.view:selected {
    outline-style: dotted;
    outline-color: @default_fg_normal;
    outline-width: 2px;
    outline-offset: -4px;
}
```

Thanks a lot for the work you do on Greymond. I think adding this rule to the theme would make navigation much clearer for many users.

Best regards.

**Comment by parhelion22** [comment]
Thank you for the suggestion. I've made small adjustments to outline-width and outline-offset to match the GTK2 theme and will commit the fix.
